Makefile

```diff
@@ -0,0 +1,44 @@
+# This Makefile just indirects to call gradlew
+
+SPARK_VERSION :=2.2.0
+SPARK_HOME    :=/usr/local/java/spark-${SPARK_VERSION}-bin-hadoop2.7
+
+CXX  := /usr/bin/g++ -std=c++14 -Wno-deprecated
```

Comment:
I thought we were using c++17?

Reply:
I wish.  But currently the CI machines are at gcc-4.8.3 which won't support c++17.